src/storage/file_io.c

```diff
@@ -502,6 +502,10 @@ static bool fileio_synchronize_sys_volume (THREAD_ENTRY * thread_p, FILEIO_SYSTE
 static bool fileio_synchronize_volume (THREAD_ENTRY * thread_p, FILEIO_VOLUME_INFO * vol_info_p, APPLY_ARG * arg);
 static int fileio_cache (VOLID volid, const char *vlabel, int vdes, FILEIO_LOCKF_TYPE lockf_type);
 static void fileio_decache (THREAD_ENTRY * thread_p, int vdes);
+void fileio_cache_dump (void);
+static void fileio_dump_cached_file (VOLID volid, int vdes, const char *vlabel);
```

Comment:
Already declared in the header.
```suggestion
static void fileio_dump_cached_file (VOLID volid, int vdes, const char *vlabel);
```

Reply:
Removed now